rotkehlchen/externalapis/opensea.py

```diff
@@ -216,6 +216,8 @@ def _deserialize_nft(
             price_in_eth = max(last_price_in_eth, floor_price)
             price_in_usd = price_in_eth * eth_usd_price
             token_id = entry['asset_contract']['address'] + '_' + entry['token_id']
+            if entry['asset_contract']["asset_contract_type"] == 'semi-fungible':
```

Comment:
Please use single quotes

```suggestion
            if entry['asset_contract']['asset_contract_type'] == 'semi-fungible':
```